spec/features/visit_requests/cancel_spec.rb

```diff
@@ -8,8 +9,16 @@
     visit "/events/#{event.slug}"
   end
 
-  it 'removes visit request from database' do
-    expect { click_link 'Cancel attendance'}.to change{VisitRequest.count}.by(-1)
-    expect(page).to have_link 'Attend'
+  it 'changes visit request status to canceled' do
+    expect { click_link 'Cancel attendance'}.to change{ visit_request.reload.status }.to (VisitRequest::CANCELED.to_s)
+    expect(page).to have_content 'Pity.. Hope to see you next time!'
```

Comment:
Use here `I18n.t('flash.visit_requests.destroy.success')`, because if you will change translation - you will need to change this test too. This is not right